README.md

```diff
@@ -72,7 +72,9 @@ optional arguments:
 ```
 
 ### Examples
-In order to create eclipse project files for a ros package called **my_pkg** that lives in a 
+
+#### Single ROS packages
```

Comment:
I'd call it `catkin` package because catkin works entirely without ROS and so does _eclipsify_.

Reply:
I agree